script.js

```diff
@@ -21,4 +21,187 @@ function closeScreen() {
+            </div>
+            
+            <div class="modal-bottom">
+                <p>${popData[data].Description}</p>
+            </div>
+            <div class="buttons-1">
+                <button class="modal-11" onclick="window.location.href='${popData[data].ProjectLink}'">See live <img src="${popData[data].liveIcon}" alt="live logo icon"></button>
+                <button class="modal-12" onclick="window.location.href='${popData[data].GitHubLink}'">See Source <img src="${popData[data].githubIcon}" alt="github logo"></button>
+            </div>
+        </div>
+    </div>
+  `;
+  // Adding event listeners to respective buttons
+
+  const modalCloseButton = document.querySelector('.modal-top');
+  function toggleModal() {
+    if (getComputedStyle(modal).display === 'flex') {
+      modal.classList.add('modal-hide');
+      setTimeout(() => {
+        modal.style.display = 'none';
+        modal.classList.remove('modal-show', 'modal-hide');
+      }, 200);
+    } else {
+      modal.style.display = 'flex';
+      modal.classList.add('modal-show');
+    }
+  }
+
+  modalCloseButton.addEventListener('click', () => toggleModal());
+
+  modal.addEventListener('click', (event) => {
+    if (event.currentTarget === event.target) toggleModal();
+  });
+}
+
+// Adding work cards dynamically
+const workProfile = document.querySelector('.work-profiles-10');
+
+for (let i = 0; i < 3; i += 1) {
+  workProfile.innerHTML += `
+  <div class="work-profile-1 ${popData[i].classNO}">
+  <div class="works-text">
+      <div class="works-title">
+          <h3>
+          Multi-Post Stories<br>
+          Gain+Glory
+          </h3>
+      </div>
+      <div class="works-tags">
+          <ul class="tags">
+              <li class="tech-tag">Ruby on Rails</li>
+              <li class="tech-tag">CSS</li>
+              <li class="tech-tag">Javascript</li>
+              <li class="tech-tag">HTML</li>
+          </ul>
+      </div>
+      <div class="works-cta">
+          <button type="submit" class="project-button" onclick="modalSection(${i})">
+              See Project
+          </button>
+      </div>
+  </div>
+</div>
+  `;
+}
+
+const workProfile2 = document.querySelector('.work-profiles-20');
+
+for (let i = 3; i < popData.length; i += 1) {
+  workProfile2.innerHTML += `
+  <div class="work-profile-1 ${popData[i].classNO}">
+  <div class="works-text">
+      <div class="works-title">
+          <h3>
+          Multi-Post Stories<br>
+          Gain+Glory
+          </h3>
+      </div>
+      <div class="works-tags">
+          <ul class="tags">
+              <li class="tech-tag">Ruby on Rails</li>
+              <li class="tech-tag">CSS</li>
+              <li class="tech-tag">Javascript</li>
+              <li class="tech-tag">HTML</li>
+          </ul>
+      </div>
+      <div class="works-cta">
+          <button type="submit" class="project-button" onclick="modalSection(${i})">
+              See Project
+          </button>
+      </div>
```

Comment:
Hey @BenMukebo, can you please clarify what you want me to do here? I cannot comprehend it and everything seems to be working fine on the website in the recent works section.

Reply:
Hello @TayyabKM. The previous reviewer is suggesting you remove the hard-coded title of the project, languages use to build the project because all of these details must be fetched from the array you created as it is one of the project's requirements. Kindly make sure to change that in order to meet all requirements. :wink: